### PR TITLE
Improve some tests

### PR DESCRIPTION
**Pull request by Gnuk**

### CODE REVIEW

File: test/cli/create-component.spec.ts [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 const expectAssetCreatedFor = (folderPath: string, componentName: string) => {
-  expect(fs.statSync(path.resolve(FAKE_DIR, `${folderPath}`)).isDirectory()).toBeTruthy();
-  expect(fs.statSync(path.resolve(FAKE_DIR, `${folderPath}/${componentName}.md`)).isFile()).toBeTruthy();
-  expect(fs.statSync(path.resolve(FAKE_DIR, `${folderPath}/${componentName}.render.pug`)).isFile()).toBeTruthy();
-  expect(fs.statSync(path.resolve(FAKE_DIR, `${folderPath}/${componentName}.mixin.pug`)).isFile()).toBeTruthy();
-  expect(fs.statSync(path.resolve(FAKE_DIR, `${folderPath}/${componentName}.code.pug`)).isFile()).toBeTruthy();
-  expect(fs.statSync(path.resolve(FAKE_DIR, `${folderPath}/_${componentName}.scss`)).isFile()).toBeTruthy();
+  const pathFolderTo = pathTo(folderPath);
+  const pathStatSyncTo = (...segments: string[]) => fs.statSync(pathFolderTo(...segments));
+  const expectFile = (filename: string) => expect(pathStatSyncTo(filename).isFile()).toBe(true);
```
Comment (pascalgrimaud) [inline test/cli/create-component.spec.ts:11]: I'm not typescript expert but would it be better to use toBeTruthy() ?

File: test/cli/create-component.spec.ts [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+
+  expect(pathStatSyncTo().isDirectory()).toBe(true);
```
Comment (pascalgrimaud) [inline test/cli/create-component.spec.ts:13]: same question about using toBeTruthy()

Comment (Gnuk) [reply test/cli/create-component.spec.ts:13]: Same answer ahah

File: test/cli/create-component.spec.ts (hunk 1 above)
Comment (pascalgrimaud) [reply test/cli/create-component.spec.ts:11]: Thanks for explaination!

**Pull request merged**